src/powersoftau/README.md

```diff
@@ -0,0 +1,14 @@
+# powersoftau command
+
+Command for processing powersoftau data (output from commands in
+https:://github.com/clearmatics/powersoftau), independent of any
+specific circuit.  Main use-case is to pre-compute the evaluations of
+the Lagrange polynomails for domains of specific sizes, so that this
```

Comment:
`Lagrange polynomails` -> Lagrange polynomials

Reply:
> Command for processing powersoftau data (output from commands in https:://github.com/clearmatics/powersoftau)

So what about renaming this command `powersoftau-processing` or `process-pot` so that the binary is named according to what it really does?

Reply:
Renamed the binary as suggested.